examples/k8s-helloworld.yaml

```diff
@@ -57,31 +50,39 @@ spec:
           value: "true"
         - name: A8_CONTROLLER_URL
           value: http://$(CONTROLLER_SERVICE_HOST):$(CONTROLLER_SERVICE_PORT)
-        - name: A8_REGISTRY_URL
-          value: http://$(REGISTRY_SERVICE_HOST):$(REGISTRY_SERVICE_PORT)
         - name: A8_CONTROLLER_POLL
           value: 5s
-        - name: A8_REGISTRY_POLL
-          value: 5s
         - name: A8_SERVICE
-          value: gateway:none
+          value: gateway
```

Comment:
Same as earlier comment. I think the none was added because nginx access log was complaining.